### DISCUSSION TITLE
Sponsoring to enable additional features

### DISCUSSION THREAD

**Discussion by jrendel-rbi** [flowwer-dev/pull-request-stats#56] [Q&A] [answered]
After sponsoring we're having a hard time unlocking the extra features.  Does the sponsorship have to be public?  If we are in an org does it have to be the org account that is the sponsor or can I sponsor individually?

**Comment by manuelmhtr** [comment, marked as answer]
Hi @jrendel-rbi, 
Sorry about that problem. Yes, the fastest and easiest way is making the organization the sponsor and setting it as public (otherwise it cannot be read by the API).

However, if you prefer to keep it private OR keep the sponsorship from your personal account, just tell me the name of the organization and I'll add it to a whitelist so you can use the premium features. You can post it here or send it to my email (m@nuel.mx).

Best! and thank you very much for your support!

**Reply by jrendel-rbi** [reply]
The org is rbilabs, thanks for a great tool❤️

**Reply by manuelmhtr** [reply]
Done :D
I've added rbilabs to the list. You should be able to use the premium features now (just make sure to use `@master` or `@v2.6.1` version of the action).

Thanks again! And happy to help with any other question or hear any feedback.
